File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        Set<Integer> hs = new HashSet<>();
        for(int num: nums) {
            if(hs.contains(num)) {
                return true;
            } else {
                hs.add(num);
            }
        }

        return false;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.cpp
#include <algorithm> 

class Solution {
public:
    int longestConsecutive(vector<int>& nums) {
        int n = nums.size();
        
        unordered_set<int> seen(nums.begin(), nums.end());
        // cout << minElement << " " << maxElement << endl;

        int res = 0;
        for(const int& num: nums) {
            int nu = num;
            if(!seen.contains(nu - 1)) {
                int temp = 1;
                while (seen.contains(nu + 1)) {
                    temp += 1;
                    nu += 1;
                }

                res = max(res, temp);
            }
        }

        return res;
    }
};
